Cargo.toml

```diff
@@ -178,6 +178,7 @@ prettyplease = "0.2.25"
 project-root = "0.2.2"
 rayon = "1.10.0"
 regex = "1.11.1"
+regress = "0.10.1"
```

Comment:
Is it possible to not include another regex dependency? It's a bit heavy.

Are we able to limit the regex usage instead?

Reply:
Rust regex and JS Regex are differently. 

In the test are differnt regexs like:

- `@app/(?!(api/enums$)).*`
- `foo/(?!bar)`

which then  would fail. I can disable the test and use rust regex if you want. 
I expect that some setups will fail because of this :)

Reply:
Removed the package with this commit: https://github.com/oxc-project/oxc/pull/8076/commits/0da9b9d8030f0bdd87e3e3a5ad588638cdaea8c3

Unicode is supported by default (flag `u`)
Case-Insenstive can not be enabled by the option `case_senstive`. It must be included in the regex, see the documentaiton here: https://docs.rs/regex/1.10.4/regex/#grouping-and-flags